setup/package.json

```diff
@@ -3,8 +3,10 @@
   "version": "1.0.0",
```

Comment:
I had a coworker who was confused about the mismatch of the package.json version and the actual version of the action. So perhaps we should start actually bumping this.

It is also likely that eventually we'll need to use peerDependencies correctly and perhaps even publish this to npm or github packages.

And lastly, the churn between package-lock.json version 1 and version 2 is going to drive me nuts (this is not your fault, it's just npm being npm).

I'll need to figure out the best way to handle that until npm version 7 becomes more of a thing. I'll stick with this new package-lock.json because it's from npm v6 and that's the LTS version, but I might need to specify this somewhere or add a check to avoid every PR coming with +/- 10k LoC.

Reply:
Yep, that sounds awful.. For now, I have matched the `package.json` version with the latest tag: `1.1.5`.